listenbrainz/domain/spotify.py

```diff
@@ -173,6 +190,42 @@ def update_latest_listened_at(user_id, timestamp):
     db_spotify.update_latest_listened_at(user_id, timestamp)
 
 
+def get_access_token(code):
```

Comment:
the spotipy lib has a bug which makes it lose the scope returned by the Spotify API. the scope is needed if we want to know whether the user wants their listens imported or not. So I've used the Spotify endpoint directly here. I'll open a ticket in spotipy but the last release was in 2017, so not sure how active it is.

Reply:
Don't forget to make a ticket to remind you to look into this.

Reply:
[LB-421](https://tickets.metabrainz.org/browse/LB-421)